Cooper/ORM/DBTools.cs

```diff
@@ -14,5 +14,10 @@ public static string SanitizeString(string value)
             }
             return value.Replace("'", "").Replace("\"", "");
         }
+
+        public static bool ProcessBoolean(object value)
+        {
+            return (value.ToString() == "y") ? true : false;
```

Comment:
It would be perfect to check if value is not null and value is boolean. Otherwise return false